<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>321</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>574</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>581</width>
      <height>371</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>581</width>
      <height>371</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Wernicke Schwabacher</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>101</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>40</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>40</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>70</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status [+]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Build">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Build V1.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_password</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_username</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
